### DISCUSSION TITLE
feature request: suggest labeling by point placement distance

### DISCUSSION THREAD

**Discussion by WeissShahaf** [talmolab/sleap#1761] [Ideas]
Dear Devs,
please add an option for label suggestions, thresholded on max point displacement. 
the reasons are:
-point and instance scores can be high even for jumps in tracking.
- while displacement can be shown in the tracker seekbar. clicking on the actual frame is very hard. especially for long videos.

thanks,
Shahaf

**Comment by WeissShahaf** [comment]
any new on this one?
i'm encoutering many videos where frames with some points all the way across the frame, yet getting higher scores then those that are actually on the animal.

this one seems to be very easy to implement. a simple threshold of average distance between all points, in pixels.
thanks
shahaf

**Comment by talmo** [comment]
Quick update: This feature is a WIP in #1862.

**Reply by roomrys** [reply]
Now merged into the develop branch!

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
